packages/ketcher-core/src/domain/entities/sgroup.ts

```diff
@@ -82,6 +82,9 @@ export class SGroup {
   neiAtoms: any;
   pp: Vec2 | null;
   data: any;
+  dataArea: any;
+  atomId?: number;
```

Comment:
What `atomId` do you mean?
What is `position`?

Reply:
You are right, I don't need this, these variables would be received as a result of getContractedPosition()
Removed